pynetfilter_conntrack/conntrack_base.py

```diff
@@ -16,9 +16,12 @@ def __init__(self, subsys, subscriptions):
         self.conntrack = None
         self.handle = None
 
+	self.subsys = subsys
```

Comment:
It is used on next list (open a conntrack handler), however it is not really used in the class.

The reason is that I saw several reports of dereferencements of data when using python clib because the garbage collector lost track of a variable used in the c library and cleaned it up.
I did not analysed this carefully, but I try to make sure the garbage collector will not dereference the data by storing systematically in the class variables that are used in the c library calls.